components/example-notebook-servers/README.md

```diff
@@ -0,0 +1,139 @@
+# Example Notebook Servers
+
+**DISCLAIMER:** The Notebooks Working Group provides these Dockerfiles and
+their images as examples only, they are not tested with our CI/CD pipelines
+and are not certified to work in every situation. As such, issues related to
+these images will be dealt with in a best efforts approach. The Notebooks
+Working Group will work on officially supporting some of these Notebook
+Server images in the near future. If you do encounter a problem in one of
+these images, contributions and issue reports are greatly appreciated.
+
+## Introduction
```

Comment:
I would suggest trying to remove the passive voice in favour for an active voice. This is a general comment for the entire document.

Reply:
Yeah, I will look into this. I'm used to scientific writing so tend to default to the passive voice.